core/test/financial_templates/Liquidatable.js

```diff
@@ -113,7 +114,7 @@ contract("Liquidatable", function(accounts) {
       expirationTimestamp: expirationTimestamp,
       withdrawalLiveness: withdrawalLiveness.toString(),
       collateralAddress: collateralToken.address,
-      finderAddress: finder.address,
+      finderAddress: Finder.address,
```

Comment:
are these changes not equivalent? `finder` ins the deployed instance of the `Finder` artifact.

Reply:
Just did it this way for consistency, since this is how we do it everywhere else (now) :)